### PR TITLE
Hw01.03.21

### PR DESCRIPTION
**Pull request by diledsys**

### CODE REVIEW

File: src/main/java/com/javae_ssential/diledsys/homeWork/hw010321/worker/Main.java [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+import java.util.Scanner;
+
+public class Main {
+
+    public static void main(String[] args) {
+        String jobTitle = "";
+        String yearJoining = "";
+        String fullname = "";
+        Worker[] workers = new Worker[5];
+        Scanner reader = new Scanner(System.in);
```
Comment (ochedenis) [inline src/main/java/com/javae_ssential/diledsys/homeWork/hw010321/worker/Main.java:13]: Split the code with some space between logical blocks.

File: src/main/java/com/javae_ssential/diledsys/homeWork/hw010321/worker/Main.java [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+                try {
+                    if (!cheackData(yearJoining)) {
+                        throw new Exception("invalid date format");
+                    } else break;
+                } catch (Exception e) {
+                    System.out.println(e.getMessage());
+                }
+            }
+            workers[i] = new Worker(fullname, jobTitle, yearJoining);
+        }
```
Comment (ochedenis) [inline src/main/java/com/javae_ssential/diledsys/homeWork/hw010321/worker/Main.java:33]: Main too large. This all "for" block can be a full-fledged method.

**Review by ochedenis — CHANGES_REQUESTED** [review]

**Comment by yuriy-tsarenko** [comment]
There are no new commits between base branch 'future_diledsys' and head branch 'hw01.03.21'

**Comment by yuriy-tsarenko** [comment]
There are no new commits between base branch 'future_diledsys' and head branch 'hw01.03.21'

**Pull request closed without merging**